support-frontend/assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -176,13 +176,14 @@ function getCopy(product: SubscriptionProduct, country: CountryGroupId) {
   };
 }
 
-function SaleHeader(props: PropTypes) {
+function CampaignHeader(props: PropTypes) {
   const product: SubscriptionProduct = 'DigitalPack';
+  const copy = getCopy(product, props.countryGroupId);
   return (
     <ProductPagehero
       appearance="campaign"
-      overheading="Digital Pack subscriptions"
```

Comment:
Looks like this might now change to just `Digital Pack`.  Is that intentional?

Reply:
Good pick up Jon! Cheers.